static/jet/css/_content.scss

```diff
@@ -0,0 +1,297 @@
+@import "globals";
+
+
+/* LINKS */
+
+a, a:visited, a:hover, a:focus {
+  color: $link-color;
+  font-weight: normal;
+  text-decoration: none;
+}
+
+a:hover, a:focus {
+  color: $hover-link-color;
+}
+
+a img {
+  border: none;
+}
+
+//a.section:link, a.section:visited {
```

Comment:
@nelmandela @HawiCaesar looks good overall. Let's remove the commented code in the `static/jet/css` directory.

Reply:
Alright, we will change this.

Reply:
Also it looks like the library theme itself has the commented code [here](https://github.com/geex-arts/django-jet/blob/dev/jet/static/jet/css/_content.scss#L20)